.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: php:composer
```

```suggestion
- package-ecosystem: composer
```
If think this would be the correct value according to the documentation.
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem

---

Interesting. This conflicts. I will make the change.